### PR TITLE
grub-set-default: add page

### PR DESCRIPTION
**Pull request by johandegn**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

It resolves part of issue #8080

### CODE REVIEW

File: pages/linux/grub-set-default.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# grub-set-default
+
+> Set the default boot entry for GRUB.
+> More information: <https://manned.org/grub-set-default>.
+
+- Set the boot entry to a entry number, name or identifier:
+
+`sudo grub-set-default {{menu_entry}}`
```
Comment (blueskyson) [inline pages/linux/grub-set-default.md:8]:
```suggestion
`sudo grub-set-default {{entry_number}}`
```

File: pages/linux/grub-set-default.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+- Set the boot entry to a entry number, name or identifier for alternative boot directory:
+
+`sudo grub-set-default --boot-directory {{/path/to/directory}} {{menu_entry}}`
```
Comment (blueskyson) [inline pages/linux/grub-set-default.md:12]:
```suggestion
`sudo grub-set-default --boot-directory {{/path/to/boot_directory}} {{entry_number}}`
```

File: pages/linux/grub-set-default.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# grub-set-default
+
+> Set the default boot entry for GRUB.
+> More information: <https://manned.org/grub-set-default>.
+
+- Set the boot entry to a entry number, name or identifier:
```
Comment (kbdharun) [inline pages/linux/grub-set-default.md:6]:
```suggestion
- Set the boot entry to an entry number, name or identifier:
```

File: pages/linux/grub-set-default.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+`sudo grub-set-default {{menu_entry}}`
+
+- Set the boot entry to a entry number, name or identifier for alternative boot directory:
```
Comment (kbdharun) [inline pages/linux/grub-set-default.md:10]:
```suggestion
- Set the boot entry to an entry number, name or identifier for an alternative boot directory:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by blueskyson — COMMENTED** [review]
I think entry_number is more specific to show that it should be a number.

**Review by blueskyson — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**